docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -1,68 +1,189 @@
+If no existing open standard is available, effort SHOULD be put into developing one. |  | Maybe we need a standard way to forward the complaint between bodies, e.g.: City <--> Provice?
+Standards that are machine testable SHOULD be preferred over those that are not. | N/A |
+Functionality using features from a non-open standard (one that doesn’t meet the Open Source  Initiative Open Standard Requirements) MAY be provided if necessary, but only in addition to compliant features. | |
+All non-compliant standards used MUST be recorded clearly in the documentation. | N/A |
+The codebase SHOULD contain a list of all the standards used with links to where they are available. | no | Created [backend#65](https://github.com/Signalen/backend/issues/65). |
 
 ## [Use continuous integration](https://standard.publiccode.net/criteria/continuous-integration.html)
 
-* Tests not currently invoked by Travis or similar public continuous integration. (Tests are run in the Datapunt CI)
-* Running tests locally is documented in the [README](https://github.com/Amsterdam/signals/blob/master/README.md#running-the-test-suite-and-style-checks)
-* No clear integration testing or capacity testing.
-* Code coverage by tests is not published.
-* Ensure test fixtures are separated from production code.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+All functionality in the source code MUST have automated tests. | mostly | In addition to unit testing, there are end-to-end tests which run on master and release branches (frontend); there are coverage reports, results require manual download.
+Contributions MUST pass all automated tests before they are admitted into the codebase. | yes |
+Contributions MUST be small. | yes | Each pull request contains a single feature or fix, but this policy is not explicit in CONTRIBUTING.md nor explictily checked by reviewer.
+The codebase MUST have active contributors. | yes | Amsterdam & VNG
+Source code test and documentation coverage SHOULD be monitored. | no | Coverage statics are not monitored but tests will fail when coverage is below a threshold.
+Policy and documentation MAY have testing for consistency with the source and vice versa. |  |
+Policy and documentation MAY have testing for style and broken links. |  |
 
 ## [Publish with an open license](https://standard.publiccode.net/criteria/open-licenses.html)
 
-* Good.
-    * License: [Mozilla Public License Version 2.0](../../LICENSE)
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+All code and documentation MUST be licensed such that it may be freely reusable, changeable and redistributable. | Yes | [Mozilla license](https://github.com/Amsterdam/signals/blob/master/LICENSE), would be nice to add SPDX license identifiers to files.
```

Comment:
The SPDX license headers are an outstanding issue (we are checking with legal). We would probably also need for the CI to detect missing SPDX headers and fail builds if they are missing.

Reply:
CI check for SPDX would be a very welcome addition. Without the SPDX in each file, a file is not as obviously freely reusable, however the licensing of the codebase is such that it meets this requirement.

Reply:
Created an issue: https://github.com/Signalen/backend/issues/91